README.md

```diff
@@ -32,58 +32,153 @@ https://github.com/cbdr/ruby-cb-api/blob/master/lib/cb/config.rb
+
+    search_result_model = search_response.model
+
+Which encapsulates details of the search result
+
+    search_result.total_count
+    search_result.last_item_index
+    search_result.city
+    search_result.state
+    search_result.postal_code
+    search_result.search_location 
+
+As well as returning back the jobs from the search result (`Cb::Models::Job` instances)
+
+    jobs = search_result.jobs
+
+Each returned job object contains details of one job listing
+
+    jobs.each do |job|
+      puts "#{job.did}: #{job.company_name} - #{job.title}"
     end
 
-You will also get meta data regarding the search itself (helpful for pagination).
+The search response can also return you any errors which occured during the job search (`Cb::Responses::Errors`). The parsed errors will contain an `Array` of error messages. If it is not empty an error occurred during the request.
 
-    puts search.cb_response.total_pages
-    puts search.cb_response.total_count
-    puts search.cb_response.errors # Hopefully this is nil! If it's not nil, it will be an array of error messages.
+    search_response.errors.parsed
 
-Coming Soon
-================
-The way that requests are handled is being completely redone. You will now only need to instantiate a single client class and pass it a request object.
+Job Search params
+=================
 
-You may now pass a block that will be executed before and after each API call. This will provide the same information that the Observer methods do.
+When performing a job search you can provide a `Hash` of search parameters. For example
 
-    client = Cb::Client.new { |request_metadata| storage_device.add(request_metadata) }
+    search_params = { location: 'Atlanta', keyword: 'Database Admin' }
+    Cb.job.search(search_params)
 
-Or just call it without the block if you don't care about an individual call's observer info
+The search params hash can include the following keys:
 
-    client = Cb::Client.new
+* keyword
+* location
+* postedwithin
+* excludeapplyrequirments
+* orderdirection
+* orderby
+* pagenumber
+* hostsite
+* siteentity
+* countrycode
 
-Pass it a request object, and you will receive the appropriate response object back
+Job details
+===========
 
-    request = Cb::Requests::Endpoint::CallName.new( { key1: 'value1', key2: 'value2' } )
-    response = client.execute request
+`Cb::Models::Job` has many accessor methods which are used to get the details of the job
 
-Currently, this workflow only works on the following endpoints:
+If you have preformed a job search, such as
 
-* Anonymous Saved Search
-* Application
-* Application External
-* Category
-* Education
-* Company
-* User
+    Cb.job.search({ location: 'Atlanta' }).results.model.jobs.each do |job|
+      puts "Job Details: #{job.company_name} - #{job.title}"
+    end
 
-Look here for future updates on this refactor!
+The following data is available from a job
```

Comment:
./agree point to 1:1 online docs.

Are you referring to https://github.com/rranshous/ruby-cb-api ?